lib/Agrammon/Web/Routes.pm6

```diff
@@ -287,10 +287,10 @@ sub application-routes(Agrammon::Web::Service $ws) {
             }
         }
 
-        # test/implement
-        post -> LoggedIn $user, 'delete_data' {
-            request-body -> %data {
-                my $ret = $ws.delete-data($user, %data);
+        # working
+        post -> LoggedIn $user, 'delete_instance' {
+            request-body -> ( :datasetName($dataset-name), :$instance, :variablePattern($variable-pattern) )  {
+                my $ret = $ws.delete-instance($user, $dataset-name, $variable-pattern, $instance);
                 content 'application/json', $ret;
```

Comment:
Returning a plain integer (or null) here, I think. The mock does the wrapping in a `deleted`, but that isn't what the code in `Agrammon::Web::Service` really returns.

Reply:
Went for 204 and exception handling instead.